### DISCUSSION TITLE
How do i upload a file and a image in Register?

### DISCUSSION THREAD

**Discussion by iChain25** [codeigniter4/shield#1112] [Q&A] [answered]
Im trying to search how and i don't see anything that would help me, so im trying my luck here. thanks

**Comment by kikuKeii** [comment, marked as answer]
Maybe you should create Extending the Controllers to handle various parts of the authentication process.

[Check Here Extending the Controllers](https://shield.codeigniter.com/customization/extending_controllers/)
